extensions/default/src/hangingprotocols/mammoDisplaySetSelector.ts

```diff
@@ -0,0 +1,170 @@
+const priorStudyMatchingRules = [
+  {
+    // The priorInstance is a study counter that indicates what position this study is in
+    // and the value comes from the options parameter.
+    attribute: 'studyInstanceUIDsIndex',
+    from: 'options',
```

Comment:
I was looking at the options documentation, it is not clear, it says this 

```md
### `from` attribute
The from attribute allows getting the attribute to test from some other object
such as the prior study, the list of studies overall or another module provided
value.  Some of the possible attributes are:

* `prior`: To get the value from the prior study.
* `activeStudy`: To match the active study
* `studies`: To match the list of studies to display
* `displaySets`: The display sets for the current study
* `allDisplaySets`: Alll available display sets
* `instance`: An instance from the current display set being tested
* `options`: Gets the options object itself, eg if you want a simple top level
  value.
```

But it is still not clear at first sight where the options are provided and by whom.

Reply:
Can you please update that documentation with extra details?